service/spawn.go

```diff
@@ -635,6 +635,10 @@ func (uis *UIServer) modifyVolume(w http.ResponseWriter, r *http.Request) {
 			uis.LoggedError(w, r, http.StatusBadRequest, errors.Errorf("host '%s' does not exist", *updateParams.HostID))
 			return
 		}
+		if vol.AvailabilityZone != h.Zone {
```

Comment:
should this be enforced on the client side also?
For example, the list of hosts to mount on could exclude hosts in other AZs.

Reply:
I thought about excluding hosts in other AZs, but I thought that might be confusing, since users don't necessarily know about this rule and might be very frustrated that their host isn't showing up. I thought it would be better to allow it and then tell them why it's restricted.

Reply:
that's a good point.